mingw-w64-cninja/PKGBUILD

```
@@ -0,0 +1,55 @@
+# Maintainer: Jean-Michaël Celerier <jeanmichael.celerier at gmail dot com>
+_realname=cninja
+pkgbase=mingw-w64-${_realname}
+pkgname=${MINGW_PACKAGE_PREFIX}-${_realname}
+pkgver=3.5.0
+release_tag=v3.5.0
+pkgrel=1
+pkgdesc="cninja, an opinionated cmake config manager"
+arch=('any')
+url="https://github.com/jcelerier/cninja"
+license=('AGPLv3')
+depends=(
+    "${MINGW_PACKAGE_PREFIX}-cmake" 
+    "${MINGW_PACKAGE_PREFIX}-clang" 
+    "${MINGW_PACKAGE_PREFIX}-lld" 
+    "${MINGW_PACKAGE_PREFIX}-ninja"
+    "${MINGW_PACKAGE_PREFIX}-libc++"
+)
+makedepends=(
+    "${MINGW_PACKAGE_PREFIX}-boost"
+)
+optdepends=()
+provides=("$pkgname=$pkgver")
+conflicts=()
+replaces=()
+source=("https://github.com/jcelerier/cninja/releases/download/v${pkgver}/cninja-${pkgver}-src.tar.gz")
+sha512sums=("19a07699e19c9b7259d869d4fbbf3a8cfc27598b3714b79553f60eb5e8f96028a743fbf42b4cb550d388be3b38348c9ad8f09e0cd849480f6330d940a188f625")
+
+build() {
+  cd "$srcdir"
+
+  mkdir -p "$srcdir/build"
+  cd "$srcdir/build"
```

```suggestion
  mkdir -p "$srcdir/build"
  cd "$srcdir/build"
```
you could remove these lines

---

Build dir should differ between both architectures, like in https://github.com/msys2/MINGW-packages/blob/7f9ec6c4789ee12e1a2e6e7fc8396cb09baea105/mingw-w64-binutils/PKGBUILD#L67

---

done